src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php

```diff
@@ -44,4 +48,44 @@ public function unsupportedMessagesProvider(): iterable
         yield [new ChatMessage('Hello!')];
         yield [$this->createMock(MessageInterface::class)];
     }
+
+    public function messagesProvider(): iterable
+    {
+        yield 'without a slash' => ['hello'];
+        yield 'including a slash' => ['hel/lo'];
+    }
+
+    /**
+     * @group time-sensitive
+     *
+     * @dataProvider messagesProvider
+     */
+    public function testValidSignature(string $message)
+    {
+        $notification = new Notification($message);
+        $adminRecipient = new AdminRecipient('foo@bar', '0611223344');
+
+        $smsMessage = SmsMessage::fromNotification($notification, $adminRecipient);
```

Comment:
```suggestion
        $smsMessage = new SmsMessage(.....);
```

This should be enough, no need to deal with a `Notification object`